### PR TITLE
refactor: Sync source code with external modules

### PR DESCRIPTION
**Pull request by o-kopysov**
# Pull Request

## Description

Current PR includes changes caused by code sync with the internal module.
After completion of the internal test, a new tag, v1.4.1, will be assigned.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [X] Test Coverage update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] My code meets the required code coverage for lines (90% and above)
- [X] My code meets the required code coverage for branches (80% and above)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by t-naumenko — APPROVED** [review]
LGTM

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -28,6 +28,9 @@ lpvs.cores=8
 lpvs.attempts=4
 
 # DB Configuration
+# The name of DB schema
+lpvs.schema=lpvs
```
Comment (tiokim) [inline src/main/resources/application.properties:32]: Do we need to add this to the `docker-compose.yml' file?

Comment (o-kopysov) [reply src/main/resources/application.properties:32]: In `docker-compose.yml` we use `database_dump.sql` for DB setup. The name of the schema is `lpvs`.
So, I added default values for `lpvs.schema` in the source code and we don't need this property in `docker-compose.yml`.

**Review by tdrozdovsky — APPROVED** [review]

**Review by m-rudyk — APPROVED** [review]
Approved

**Pull request merged**